### PR TITLE
Mesh and rope docs, typos

### PR DESCRIPTION
**Pull request by alvinsight**

### CODE REVIEW

**Comment by GoodBoyDigital** [comment]
good stuff thanks @alvinsight !

File: src/deprecation.js [hunk 1]
```diff
@@ -16,6 +17,16 @@ core.AssetsLoader = function () {
     window.console.warn("The loader system was overhauled in pixi v3, please see the new PIXI.Loader class.");
 };
 
+core.DisplayObjectContainer = function () {
+    window.console.warn("The Strip class has been renamed to Mesh, please use Container from now");
```
Comment (englercj) [inline src/deprecation.js]: Pretty sure that is a typo!

File: src/extras/mesh/Mesh.js [hunk 2]
```diff
@@ -384,10 +378,10 @@ Mesh.prototype.getBounds = function (matrix)
  * @static
  * @constant
  * @property {object} DrawModes
- * @property {number} DrawModes.TRIANGLE_Mesh
+ * @property {number} DrawModes.TRIANGLE_MESH
  * @property {number} DrawModes.TRIANGLES
  */
 Mesh.DrawModes = {
```
Comment (englercj) [inline src/extras/mesh/Mesh.js]: `DrawModes` looks like a class, can we change it to `DRAW_MODES`?

**Comment by englercj** [comment]
:+1: Looks awesome dude, keep up the great work!

File: src/extras/mesh/Mesh.js (hunk 2 above)
Comment (alvinsight) [inline src/extras/mesh/Mesh.js]: Yes, you're right we can, added in #1486

**Pull request merged**